oop_two/src/init/run/RUN_Car.java

```diff
@@ -0,0 +1,49 @@
+package init.run;
+
+import init.run.company.details.Engine;
+import init.run.company.professions.Driver;
+import init.run.company.professions.Person;
+import init.run.company.vehicles.Car;
+import init.run.company.vehicles.type.Lorry;
+import init.run.company.vehicles.type.SportCar;
+
+public class RUN_Car {
+    public static void main(String[] args) {
+        Person driverC = new Driver("Ed E.X.", 3);
+        Person driverL = new Driver("Max R.A.", 5);
+        Person driverS = new Driver("Zak S.W.", 7);
+
+        Engine engineC = new Engine(240.9, "honda");
+        Engine engineL = new Engine(932.2, "kraz");
+        Engine engineS = new Engine(725.4, "dodge");
+
+        Car carC = new Car("Honda Accord", "light", 1321.42, driverC, engineC);
+        Car carL = new Lorry("Belaz", "lorry", 5441.32, driverL, engineL, 8500);
+        Car carS = new SportCar("McLaren", "sport car", 982.83, driverS, engineS, 320);
+
+        System.out.println("================ GET INFO ================");
+        System.out.println(carC);
+        System.out.println("================ Drive ================");
+        engineC.start();
```

Comment:
task says that Car class should have have start and stop methods. consider to implement this methods in Car class which will use methods of Engine class inside of the Car, not outside